app/views/projects/members/index.html.erb

```diff
@@ -16,6 +16,25 @@
       <div class="shadow">
         <%= turbo_frame_tag("member-update-alert") %>
         <table class="min-w-full table-fixed dark:divide-gray-600">
+          <thead>
+            <tr
```

Comment:
**chore (blocking):** Same comment here about the table header.  Would probably be really easy to make a re-usable component for this.  Let me know if you want me do to that and I will make it and get it in to main so you can use it

Reply:
As per the comment above we will update this in a different pr to create a table header component

Reply:
Sounds good